doc/jointcal.tex

```diff
@@ -29,527 +43,603 @@
 \def\bf{\normalfont\bfseries}
 \def\rm{\normalfont \rmfamily}
 
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\title{Documentation of the astrometry component of the jointcal package}
-%\subtitle{(V0)}
-\author{Pierre Astier}
-\affil{LPNHE/IN2P3/CNRS (Paris)}
-
+\makeatother
 
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \begin{document}
 
-\maketitle
-
-\abstract{The jointcal package aims at optimizing simultaneously
```

Comment:
Suggest: "The jointcal package simultaneously optimizes... images.  In principle and often in practice, this approach produces..."